Introduce new attribute for RH Docs and fixes (#2917)
theforeman/foreman-documentation: guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc

====
+
ifdef::satellite[]

> For more information on host configuration properties, see {RHELDocsBaseURL}8/html-single/configuring_and_managing_identity_management/index#con_host-entry-LDAP_managing-hosts-ui[Host entry in IdM LDAP] in _Configuring and managing Identity Management_.

Reviewer: ```suggestion
For more information on host configuration properties, see {RHELDocsBaseURL}8/html-single/configuring_and_managing_identity_management/index#con_host-entry-LDAP_managing-hosts-ui[Host entry in IdM LDAP] in _{RHEL}{nbsp}8 Configuring and managing Identity Management_.
```

----
+
ifdef::satellite[]

> For more information on managing services, see {RHELDocsBaseURL}9/html/accessing_identity_management_services/index[Accessing Identity Management services] in _{RHEL} guide_.

Reviewer: ```suggestion
For more information on managing services, see {RHELDocsBaseURL}9/html/accessing_identity_management_services/index[_{RHEL}{nbsp}9 Accessing Identity Management services_].
```

> For more information on managing services, see {RHELDocsBaseURL}9/html/accessing_identity_management_services/index[Accessing Identity Management services] in the _{RHEL} guide_.

Reviewer: ```suggestion
For more information on managing services, see {RHELDocsBaseURL}9/html/accessing_identity_management_services/index[_{RHEL}{nbsp}9 Accessing Identity Management services_].
```
